lldp_dcbx_nl.c

```diff
@@ -279,16 +279,17 @@ static int get_state(char *ifname, __u8 *state)
 		NLMSG_ALIGN(sizeof(struct dcbmsg)));
 
 	if (d->cmd != DCB_CMD_GSTATE) {
-		return -EIO;
+		rval = -EIO;
+		goto err_out;
 	}
 	if (rta->rta_type != DCB_ATTR_STATE) {
 		rval = -EIO;
 	} else {
 		*state = *(__u8 *)NLA_DATA(rta);
 	}
 
+err_out:
```

Comment:
Minor nitpick, as this is both the successful return path as well as the error return path, the label `err_out` is a little misleading. I would just call it `out`, but otherwise, it looks functionally correct.

This convention is already used elsewhere in the code.